<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainModelUI</class>
 <widget class="QWidget" name="TrainModelUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainModelUI</string>
  </property>
  <widget class="QLabel" name="trainWeightButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Train Weight</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="trainLengthButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Train Length</string>
   </property>
  </widget>
  <widget class="QLabel" name="passengersOnTrain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># of Passengers</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentCapacity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>% Capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentTemp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Temperature (F)</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentMode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Mode</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>21</width>
     <height>191</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="trainWeightV">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="trainLengthValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="passengersOnTrainValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentCapacityValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentTempValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>75F</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentModeValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="currentSpeedLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentAccelerationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Acceleration</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentPowerLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Power Draw</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentSpeedValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 m/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentAccelerationValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 m/s/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentPowerValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0V</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>21</width>
     <height>191</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleLights">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn On/Off Train Lights</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputPowerConfirm">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleDoorLeft">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Open/Close Left Doors</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPowerResult">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="currentVelocityLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Power</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Module</string>
   </property>
  </widget>
  <widget class="QLabel" name="lightStateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Light State</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftDoorStateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Door State</string>
   </property>
  </widget>
  <widget class="QLabel" name="lightStateValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftDoorStateValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSED</string>
   </property>
  </widget>
  <widget class="QLabel" name="crewMembers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># of Crew Members</string>
   </property>
  </widget>
  <widget class="QLabel" name="crewMembersValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emergencyBrake">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Emergency Brake</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>Signal Pickup Failure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0)</string>
   </property>
   <property name="text">
    <string>Power Failure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Train Engine Failure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Failure Modes</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightDoorStateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Door State</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightDoorStateValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleDoorRight">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Open/Close Right Doors</string>
   </property>
  </widget>
  <action name="actiontrainWeight">
   <property name="text">
    <string>trainWeight</string>
   </property>
  </action>
  <zorder>listView</zorder>
  <zorder>trainWeightButton</zorder>
  <zorder>trainLengthButton</zorder>
  <zorder>passengersOnTrain</zorder>
  <zorder>percentCapacity</zorder>
  <zorder>currentTemp</zorder>
  <zorder>currentMode</zorder>
  <zorder>line</zorder>
  <zorder>trainWeightV</zorder>
  <zorder>trainLengthValue</zorder>
  <zorder>passengersOnTrainValue</zorder>
  <zorder>percentCapacityValue</zorder>
  <zorder>currentTempValue</zorder>
  <zorder>currentModeValue</zorder>
  <zorder>tableWidget</zorder>
  <zorder>currentSpeedLabel</zorder>
  <zorder>currentAccelerationLabel</zorder>
  <zorder>currentPowerLabel</zorder>
  <zorder>currentSpeedValue</zorder>
  <zorder>currentAccelerationValue</zorder>
  <zorder>currentPowerValue</zorder>
  <zorder>line_2</zorder>
  <zorder>toggleLights</zorder>
  <zorder>inputPowerConfirm</zorder>
  <zorder>toggleDoorLeft</zorder>
  <zorder>inputPowerResult</zorder>
  <zorder>currentVelocityLabel</zorder>
  <zorder>label</zorder>
  <zorder>lightStateLabel</zorder>
  <zorder>leftDoorStateLabel</zorder>
  <zorder>lightStateValue</zorder>
  <zorder>leftDoorStateValue</zorder>
  <zorder>crewMembers</zorder>
  <zorder>crewMembersValue</zorder>
  <zorder>emergencyBrake</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_2</zorder>
  <zorder>rightDoorStateLabel</zorder>
  <zorder>rightDoorStateValue</zorder>
  <zorder>toggleDoorRight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
